qt-box-editor.pro

```diff
@@ -1,5 +1,5 @@
 TEMPLATE = app
-VERSION = 1.12dev
+VERSION = 1.13dev
```

Comment:
Right, this is just a configuration file. Not necessary to increase the version number.